src/Lykke.Common.Validation/ImageType/ImageType.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections.Generic;
+
+namespace Lykke.Common.Validation.ImageType
+{
+    public class ImageType
+    {
+        public IEnumerable<string> Extensions { get; set; }
+
+        public IEnumerable<string> HexSignatures { get; set; }
+
+        #region Predefined Image types
+
+        public static readonly ImageType JpgImageType = new ImageType
+        {
+            HexSignatures = new List<string> {"FFD8FFDB", "FFD8FFE0", "FFD8FFE1"},
+            Extensions = new List<string> {".jpg", ".jpeg"}
+        };
+
+        public static readonly ImageType PngImageType = new ImageType
+        {
+            HexSignatures = new List<string> {"89504E470D0A1A0A"},
```

Comment:
I think it is better to store byte signature. It eliminates all conversion from string to byte in validators

Reply:
I agree, changed signatures to byte array.